code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -140,6 +140,64 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/cheeseburger
 
+/datum/recipe/microwave/appendixburger
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/organ/internal/appendix
+	)
+	result = /obj/item/reagent_containers/food/snacks/appendixburger
+
+/datum/recipe/microwave/appendixburger_bitten
+	items = list(
+		/obj/item/reagent_containers/food/snacks/appendix,
+		/obj/item/reagent_containers/food/snacks/bun
+		)
+	result = /obj/item/reagent_containers/food/snacks/appendixburger
+
+/datum/recipe/microwave/bearger
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/monstermeat/bearmeat
+	)
+	result = /obj/item/reagent_containers/food/snacks/bearger
+
+/datum/recipe/microwave/fivealarmburger
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/grown/ghost_chili,
+		/obj/item/reagent_containers/food/snacks/grown/ghost_chili
+		)
+	result = /obj/item/reagent_containers/food/snacks/fivealarmburger
+
+/datum/recipe/microwave/baconburger
+	items = list(
```

Comment:
Any reason why it requires so much bacon and cheesewedge? I have been eating bacon burgers the wrong way my entire life?

Reply:
Just straight ported from tg, can reduce to whatever is reasonable

Reply:
Ask what others think about this amount, i am not that knowledgeable with recipes to know if its too much.

Reply:
For a burger like this, I'd probably knock it down to two pieces of bacon and one cheese wedge

Reply:
doneskis